### DISCUSSION TITLE
Do you plan to add `writeData` method for Google Fit

### DISCUSSION THREAD

**Discussion by marcinkornek** [doville/rn-fitness-tracker#103] [General]
@alarm109 Great library, thanks for your work 💪 
I tried out Apple Health and Google Fit integration in your example app and some methods are missing in Google Fit. Based on documentation for Google Fit: "Easily write and read data such as steps, vitals, workouts, and more." but it's not possible to write data other than "writeWorkouts" right now 😢 
Do you plan to add `writeData` method (and maybe `writeDataArray`) to Google Fit integration for writing eg. weight?

**Comment by alarm109** [comment]
Hi @marcinkornek,
Thank you for using our package and writing to us.

Yes, we are currently working on making a method for writing `restricted access data types` data to googleFit. I think weight will be the first data type. I will mention you here when the feature update is released.
